.github/workflows/cmake-multi-platform.yml

```
@@ -120,31 +92,28 @@ jobs:
       # Build your program with the given configuration. Note that --config is needed because the default Windows generator is a multi-config generator (Visual Studio generator).
-      run: cmake --build ${{ steps.strings.outputs.build-output-dir }} --config ${{ matrix.build_type }}
+      run: cmake --build build/ --config Release
+
+    - name: Package (CPack)
+      # Build your program with the given configuration. Note that --config is needed because the default Windows generator is a multi-config generator (Visual Studio generator).
+      run: cmake --build build/ --target package
 
     - name: Test
-      if: ${{ matrix.os == 'ubuntu-latest' && matrix.c_compiler != 'mingw-w64-gcc' }}
+      if: ${{ matrix.platform.os == 'ubuntu-latest' && matrix.platform.c_compiler != 'mingw-w64-gcc' }}
       working-directory: ${{ steps.strings.outputs.build-output-dir }}
```

actionlint (1.7.7)

property "strings" is not defined in object type {}

(expression)